content/docs/guides/job-scheduling.md

```diff
@@ -0,0 +1,101 @@
++++
+title = "Job Scheduling"
+description = "Schedule job with gang-scheduling"
+weight = 10
+toc = true
+[menu.docs]
+  parent = "guides"
+  weight = 30
++++
+
+## Running jobs with gang-scheduling
+To use gang-scheduling, you have to install kube-arbitrator in your cluster first as a secondary scheduler of Kubernetes and configure operator to enable gang-scheduling. 
+
+* Kube-arbitrator's introduction is [here](https://github.com/kubernetes-incubator/kube-arbitrator), and also check how to install it [here](https://github.com/kubernetes-incubator/kube-arbitrator/blob/master/doc/usage/tutorial.md).
+* Take tf-operator for example, enable gang-scheduling in tf-operator by setting true to `--enable-gang-scheduling` flag.
+
+**Note:** Kube-arbitrator and operator in Kubeflow achieve gang-scheduling by using pdb. operator will create the pdb of the job automatically. You can know more about pdb [here](https://kubernetes.io/docs/tasks/run-application/configure-pdb/).
+
+To use kube-arbitrator to schedule your job as a gang, you have to specify the schedulerName in each replica; for example.
+
+```yaml
+apiVersion: "kubeflow.org/v1alpha2"
+kind: "TFJob"
+metadata:
+  name: "tfjob-gang-scheduling"
+spec:
+  tfReplicaSpecs:
+    Worker:
+      replicas: 1
+      template:
+        spec:
+          schedulerName: kube-batchd
+          containers:
+          - args:
+            - python
+            - tf_cnn_benchmarks.py
+            - --batch_size=32
+            - --model=resnet50
+            - --variable_update=parameter_server
+            - --flush_stdout=true
+            - --num_gpus=1
+            - --local_parameter_device=cpu
+            - --device=gpu
+            - --data_format=NHWC
+            image: gcr.io/kubeflow/tf-benchmarks-gpu:v20171202-bdab599-dirty-284af3
+            name: tensorflow
+            resources:
+              limits:
+                nvidia.com/gpu: 1
+            workingDir: /opt/tf-benchmarks/scripts/tf_cnn_benchmarks
+          restartPolicy: OnFailure
+    PS:
+      replicas: 1
+      template:
+        spec:
+          schedulerName: kube-batchd
+          containers:
+          - args:
+            - python
+            - tf_cnn_benchmarks.py
+            - --batch_size=32
+            - --model=resnet50
+            - --variable_update=parameter_server
+            - --flush_stdout=true
+            - --num_gpus=1
+            - --local_parameter_device=cpu
+            - --device=cpu
+            - --data_format=NHWC
+            image: gcr.io/kubeflow/tf-benchmarks-cpu:v20171202-bdab599-dirty-284af3
+            name: tensorflow
+            resources:
+              limits:
+                cpu: '1'
+            workingDir: /opt/tf-benchmarks/scripts/tf_cnn_benchmarks
+          restartPolicy: OnFailure
+```
+
+### About kube-arbitrator and gang-scheduling
```

Comment:
Let's make this heading a level 2 rather than a level 3, to make the doc's table of contents easier to read.